lib/common/util/config.js

```diff
@@ -97,7 +97,9 @@ function initializeConfig () {
 
   const languages: Array<MessageFile> = [
     // $FlowFixMe - assume file exists and make flow happy
-    require('../../../i18n/english.yml')
+    require('../../../i18n/english.yml'),
+    // $FlowFixMe - assume file exists and make flow happy
```

Comment:
Same issue here as above -- I'm weary that this PR will add the polish translation when that should've been done in #735